Lab 1 "External (file) sorts"/Lab 1 "External (file) sorts"/main.cpp

```diff
@@ -0,0 +1,60 @@
+//
+//  main.cpp
+//  Lab 1 "External (file) sorts"
+//
+//  Created by Antony Miroshnichenko on 20.02.2024.
+//
+
+
+
+#include "Funcs.hpp"
+
+enum SortTypes {Прямая_Сортировка = 0, Естественная_Сортировка = 1};
+
+
+
+int main()
+{
+    
+    const int P = Естественная_Сортировка;
+    
+    std::string fileName = "A.txt";
+    const int numbersCount = 10;
+    const int maxNumberValue = 100000;
+    
+    switch (P) {
+        case 0:
```

Comment:
Какой толк от перечисления, если вы дальше его не используете?
```suggestion
    switch (sortType ) {
        case SortType::DirectMerge:
```